52.md

```diff
@@ -35,20 +35,19 @@ The list of tags are as follows:
 * `t` (optional, repeated) hashtag to categorize calendar event
 * `r` (optional, repeated) references / links to web pages, documents, video calls, recorded videos, etc.
 
-The following tags are deprecated:
-* `name` name of the calendar event. Use only if `title` is not available.
-
 ```json
 {
   "id": <32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>,
   "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
   "created_at": <Unix timestamp in seconds>,
   "kind": 31922,
-  "content": "<description of calendar event>",
+  "content": "",
```

Comment:
Won't description go here anymore? `""` alone might imply no description is allowed.

Reply:
I got confused.